README.md

```diff
@@ -12,6 +12,20 @@ Reusable foundation for java RDF applications that provide a web UI or web API o
 
 ## Changelog
 
+**4.0.0**
+   * Appbase is compiled by JDK 21 and is compatible only with Java versions 21 and above.
+   * This version upgrades the following dependencies with major impact.
+     This may cause backwards compatibility issues for applications using Appbase:
+     * Tomcat 9.0 -> 11.0
+     * Jena 3.9 -> 5.6
+     * Lib 3.1 -> 4.0
+     * Jersey 2.25 -> 3.1
+   * Replaced all usages of javax servlet infrastructure with jakarta.
+   * Maven artifacts for Appbase version 4.0.0 and onwards are published to the Epimorphics public S3 Maven repository.
+   * The `DatasetAccessor` interface from Jena 3 is replaced by `com.epimorphics.appbase.data.DatasetAccessor`:
+     * It is directly implemented by `RDFConnectionDatasetAccessor` and indirectly by various wrapper classes which are unchanged from the previous version.
+     * It has removed the `containsModel` method since no Jena 5 equivalent exists. Instead, use `getModel` to obtain the model, then `isEmpty` on the model.
```

Comment:
Don't think that's a viable workaround for a large model but can find any uses of it downstream so not a show stopper. Suggest removing the second sentence.